lib/fog/vsphere/compute.rb

```diff
@@ -329,7 +330,7 @@ def self.data
                  "tools_state"      => "toolsOk",
                  "name"             => "jefftest",
                  "operatingsystem"  => "Red Hat Enterprise Linux 6 (64-bit)",
-                 "path"             => "/",
+                 "path"             => "/Solutions/wibble",
```

Comment:
This is in a mock. The `path` was not used by any other tests and I wanted to have a VM that was in a folder so the folder test works correctly. If you prefer, I could add another VM to the mocks instead of editing an existing one.